### DISCUSSION TITLE
Plan to keep synchronized with GOLD/JGI

### DISCUSSION THREAD

**Discussion by aclum** [microbiomedata/issues#1035] [General]
Document needs and create a plan to keep synchronized with GOLD/JGI. 
-use cases
 Changes in gold (ie deletion of ids, updates to correct errors in metadata

TO DO: 
-How to define which organization is the source of truth
-determine update frequency,

**Comment by turbomam** [comment]
This might actually be three (or higher) way monitoring/reporting/synchronization 
- upstream source like NEON
- NCBI Biosample
- GOLD
- NMDC

Here's an example where the environmental triad values are inconsistent: 
- https://gold.jgi.doe.gov/biosample?id=Gb0316050
- https://www.ncbi.nlm.nih.gov/biosample/SAMEA7723608
